specification/network/resource-manager/Microsoft.Network/stable/2022-09-01/networkWatcher.json

```diff
@@ -3431,6 +3453,46 @@
           "readOnly": true,
           "type": "integer",
           "format": "int32"
+        },
+        "nextHopAnalysis": {
+          "$ref": "#/definitions/NextHopResult",
+          "description": "The NextHop Details from the source."
+        },
+        "sourceSecurityRuleAnalysis": {
+          "$ref": "#/definitions/NetworkConfigurationDiagnosticResponse",
+          "description": "SecurityAnalysis results at the source endpoint."
+        },
+        "destinationSecurityRuleAnalysis": {
+          "$ref": "#/definitions/NetworkConfigurationDiagnosticResponse",
+          "description": "SecurityAnalysis results at the destination endpoint."
+        },
+        "sourcePortStatus": {
+          "type": "string",
+          "description": "Port Status at the Source.",
+          "enum": [
+            "NextHop",
```

Comment:
PortStatus enum is not this. Please check again and same for destinationPortStatus as well